config/initializers/assets.rb

```diff
@@ -5,10 +5,3 @@
 
 # Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
-# Add Yarn node_modules folder to the asset load path.
-Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
I can't tell if this config is custom and shouldn't be removed.

Reply:
Adding node_modules to the asset path used to be the thing to do but these days using import maps means it's not necessary.